src/mmda/parsers/pdfplumber_parser.py

```diff
@@ -1,15 +1,56 @@
-from typing import Optional, List, Dict, Union
-
+import itertools
 import string
+from typing import Dict, List, Optional, Union
+
 import pdfplumber
+import pdfplumber.utils as ppu
+from pdfplumber._typing import T_num
 
-import itertools
-from mmda.types.span import Span
-from mmda.types.box import Box
+from mmda.parsers.parser import Parser
 from mmda.types.annotation import SpanGroup
+from mmda.types.box import Box
 from mmda.types.document import Document
-from mmda.parsers.parser import Parser
+from mmda.types.metadata import Metadata
 from mmda.types.names import PagesField, RowsField, SymbolsField, TokensField
+from mmda.types.span import Span
+
+
+class WordExtractorWithFontInfo(ppu.WordExtractor):
+    """Override WordExtractor methods to append additional char-level info."""
+
+    def __init__(
+        self,
+        x_tolerance: T_num = ppu.DEFAULT_X_TOLERANCE,
+        y_tolerance: T_num = ppu.DEFAULT_Y_TOLERANCE,
```

Comment:
T_num is `int`, right? we could annotate with that and avoid the risky import from the private class.